### PR TITLE
494 block new applications

### PR DESCRIPTION
**Pull request by marcellmueller**
https://app.zenhub.com/workspaces/ccbc---delivery-board-61c0c1204c8bcb001491c5f3/issues/bcgov/conn-ccbc-portal/494

The custom `create_application` mutation is being handled in #566 now, I've updated the tickets accordingly.

### CODE REVIEW

File: app/pages/dashboard.tsx [hunk 1]
```diff
@@ -89,17 +98,25 @@ const Dashboard = ({ preloadedQuery }: RelayProps<{}, dashboardQuery>) => {
...
-          <p>There are no currently open intakes.</p>
+          <StyledAlert size="small" variant="warning">
+            <p>
+              New applications will be accepted after updates to ISED&lsquo;s
+              Eligibility Mapping tool are released.
+            </p>
+            <p>
+              Please check this page after <b>September 15</b> for an update.
+            </p>
+          </StyledAlert>
```
Comment (matthieu-foucault) [inline app/pages/dashboard.tsx:109]: Wanna put this in a component to reuse in both pages?

File: app/tests/pages/index.test.tsx [hunk 2]
```diff
@@ -39,4 +58,26 @@ describe('The index page', () => {
...
     expect(screen.getByText('Go to dashboard')).toBeInTheDocument();
   });
+
+  it('Does not display alert message when there is an open intake', async () => {
+    pageTestingHelper.loadQuery();
+    pageTestingHelper.renderPage();
+
+    expect(
+      screen.queryByText(
+        `New applications will be accepted after updates to ISED‘s Eligibility Mapping tool are released.`
```
Comment (matthieu-foucault) [inline app/tests/pages/index.test.tsx:68]: Same comment as the previous PR regarding having this text in a variable. If we're asked to change the text, this test will not fail (but it should)

**Review by matthieu-foucault — CHANGES_REQUESTED** [review]
Looks good, just a bit of code cleanup

File: app/tests/pages/index.test.tsx (hunk 2 above)
Comment (marcellmueller) [reply app/tests/pages/index.test.tsx]: Sounds good, moved the message to a constant.

**Pull request merged**